src/Pages/Help/HelpPage.jsx

```diff
@@ -0,0 +1,99 @@
+import {
+  NavBar,
+  NavBarBottom,
+  NavBarTop,
+  Button,
+  AccordionSummary,
+} from "Components";
+import { useAuth, useModal } from "Context";
+import badges from "Data/Img/badges.png";
+import "../CommonStyling.css";
+import "./HelpPage.css";
+
+export const HelpPage = () => {
+  const { setProfileMenu, authClickHandler } = useModal();
+  const {
+    authState: { token },
+  } = useAuth();
+
+  return (
+    <div className="help-page-body">
+      <NavBar />
+      <NavBarBottom />
+      <NavBarTop />
+      <div
+        className="help-page-content"
+        onClick={() => {
+          setProfileMenu(false);
+        }}
+      >
+        <div className="help-page-header">
+          <h1>FAQs</h1>
+          <div className="flex-row login-btn-desktop">
+            <Button
+              onClick={authClickHandler}
+              label={token ? "Logout" : "Login"}
+              btnClassName="btn primary-outline-btn-md"
+            />
+          </div>
+        </div>
+        <div className="help-page-faq">
+          <details>
+            <AccordionSummary
+              text="What games can I play? How many questions are there in a single
+                quiz?"
+            />
+            <div className="help-faq-text">
+              <p className="mg-1-bot">
+                Currently, We have only 8 categories, with just 3-4 questions
+                per quiz.
+              </p>
+              <ol className="list-basic">
+                <li>Avengers</li>
+                <li>Sports</li>
+                <li>Movies</li>
+                <li>Travel</li>
+                <li>Science</li>
+                <li>Technology</li>
+                <li>Animals</li>
+                <li>Language</li>
+              </ol>
```

Comment:
Suggestion: you could map the list items here.

Reply:
Noted